CHANGELOG.md

```diff
@@ -1,5 +1,367 @@
+- *(macros)* Add AST node methods for macro-related nodes ([#14492], thanks @HertzDevil)
+- *(macros)* Add macro methods for `Select` ([#14600], thanks @HertzDevil)
+- *(macros)* Add `TypeNode#private?`, `#public?` and `#visibility` ([#11696], thanks @Hadeweka)
+- *(networking)* Relax type restriction of handlers in `HTTP::Server` constructor ([#14413], thanks @hugopl)
+- *(networking)* **[security]** OpenSSL: don't change default cipher suites ([#14655], thanks @ysbaddaden)
+- *(runtime)* runtime tracing ([#14659], thanks @ysbaddaden)
+- *(system)* Add `EventLoop#run(blocking)` and `EventLoop#interrupt` ([#14568], thanks @ysbaddaden)
+- *(system)* Add `Crystal::System::Thread.current_thread?`, `#scheduler?` ([#14660], thanks @ysbaddaden)
+- *(text)* Add `Regex::MatchOptions` overload for `String#index!` ([#14462], thanks @HertzDevil)
+- *(text)* Add `StringPool#get?` ([#14508], thanks @HertzDevil)
+- *(text)* Add `pkg_config` names for `pcre2` and `pcre` ([#14584], thanks @straight-shoota)
+
+[#14401]: https://github.com/crystal-lang/crystal/pull/14401
+[#14693]: https://github.com/crystal-lang/crystal/pull/14693
+[#14486]: https://github.com/crystal-lang/crystal/pull/14486
+[#14167]: https://github.com/crystal-lang/crystal/pull/14167
+[#14532]: https://github.com/crystal-lang/crystal/pull/14532
+[#14484]: https://github.com/crystal-lang/crystal/pull/14484
+[#14514]: https://github.com/crystal-lang/crystal/pull/14514
+[#14612]: https://github.com/crystal-lang/crystal/pull/14612
+[#14459]: https://github.com/crystal-lang/crystal/pull/14459
+[#14492]: https://github.com/crystal-lang/crystal/pull/14492
+[#14600]: https://github.com/crystal-lang/crystal/pull/14600
+[#11696]: https://github.com/crystal-lang/crystal/pull/11696
+[#14413]: https://github.com/crystal-lang/crystal/pull/14413
+[#14655]: https://github.com/crystal-lang/crystal/pull/14655
+[#14659]: https://github.com/crystal-lang/crystal/pull/14659
+[#14568]: https://github.com/crystal-lang/crystal/pull/14568
+[#14660]: https://github.com/crystal-lang/crystal/pull/14660
+[#14462]: https://github.com/crystal-lang/crystal/pull/14462
+[#14508]: https://github.com/crystal-lang/crystal/pull/14508
+[#14584]: https://github.com/crystal-lang/crystal/pull/14584
+
+#### compiler
+
+- *(codegen)* Add compiler flags `-Os` and `-Oz` to optimize binary size ([#14463], thanks @ysbaddaden)
+- *(codegen)* Add compiler support for AVR architecture (Arduino) ([#14393], thanks @ysbaddaden)
+- *(semantic)* Allow rescuing exceptions that include a module ([#14553], thanks @Blacksmoke16)
+
+[#14463]: https://github.com/crystal-lang/crystal/pull/14463
+[#14393]: https://github.com/crystal-lang/crystal/pull/14393
+[#14553]: https://github.com/crystal-lang/crystal/pull/14553
+
+#### tools
+
+- *(docs-generator)* **[security]** Sanitize doc generation ([#13786], thanks @nobodywasishere)
+
+[#13786]: https://github.com/crystal-lang/crystal/pull/13786
```

Comment:
Including this seems confusing, since it was reverted.

Reply:
Yes, I have already removed it from the milestone, but the changelog needs to be regenerated.